lib/src/stores/exchange_trade/exchange_trade_store.dart

```diff
@@ -45,6 +46,8 @@ abstract class ExchangeTradeStoreBase with Store {
 
   Timer _timer;
 
+  Box<Trade> trades;
```

Comment:
Change order of definition of properties (public members of class is going first then private)